python/ray/experimental/state.py

```diff
@@ -1042,15 +1042,18 @@ def local_schedulers(self):
         Returns:
             A list of the live local schedulers.
         """
-        if self.use_raylet:
-            raise Exception("The local_schedulers() method is deprecated.")
         clients = self.client_table()
         local_schedulers = []
-        for ip_address, client_list in clients.items():
-            for client in client_list:
-                if (client["ClientType"] == "local_scheduler"
-                        and not client["Deleted"]):
+        if self.use_raylet:
```

Comment:
Do we want to keep `local_schedulers()` as part of the API? It seems somewhat redundant with `client_table()`.

Reply:
Hmm, it seems like a nice helper method to have. I suppose after all the old code is deleted this will all look nicer.